pkgs/tools/misc/glasgow/default.nix

```diff
@@ -46,24 +48,24 @@ python3.pkgs.buildPythonApplication rec {
     make -C firmware LIBFX2=${python3.pkgs.fx2}/share/libfx2
     cp firmware/glasgow.ihex software/glasgow
     cd software
-    export SETUPTOOLS_SCM_PRETEND_VERSION="${realVersion}"
+    export PDM_BUILD_SCM_VERSION="${realVersion}"
   '';
 
   # installCheck tries to build_ext again
   doInstallCheck = false;
 
   postInstall = ''
     mkdir -p $out/etc/udev/rules.d
-    cp $src/config/99-glasgow.rules $out/etc/udev/rules.d
+    cp $src/config/70-cypress.rules $out/etc/udev/rules.d
```

Comment:
My PR does not install the `cypress` rules, since i saw the currently packaged version also does not install it even though they're present in the code base. Are they needed?

Reply:
https://github.com/GlasgowEmbedded/glasgow/blob/00ad180b0a0a5d1e56f1d1be4a53fcbea4ff590b/docs/manual/src/build.rst#factory-flashing